src/styles/components/_stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  @extend %flex-between-center;
+  &__star {
+    width: 1.6rem;
+    height: 1.6rem;
+    margin-right: 4px;
+
+    @extend %bgc-image;
+
+    background-image: url('../images/star.svg');
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &.stars--4 > .stars__star:nth-child(-n + 4) {
+    background-image: url('../images/star-active.svg');
```

Comment:
The relative path for `background-image` should be written using double quotes, not single quotes, as per the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).